.gitmodules

```diff
@@ -1,12 +1,14 @@
 [submodule "Hazel/vendor/spdlog"]
 	path = Hazel/vendor/spdlog
 	url = https://github.com/gabime/spdlog
+	branch = v1.x
```

Comment:
This should be undone; this PR is for a premake change, not changing git modules.